src/encord_active/public/active_project.py

```diff
@@ -0,0 +1,77 @@
+from typing import Dict
+
+import pandas as pd
+from sqlalchemy import MetaData, Table, create_engine, select, text
+from sqlalchemy.engine import Engine
+
+
+def get_active_engine(path_to_db: str) -> Engine:
+    return create_engine(f"sqlite:///{path_to_db}")
+
+
+class ActiveProject:
+    def __init__(self, engine: Engine, project_name: str):
+        self._engine = engine
+        self._metadata = MetaData(bind=self._engine)
+        self._project_name = project_name
+
+        active_project = Table("active_project", self._metadata, autoload_with=self._engine)
+        stmt = select(active_project.c.project_hash).where(active_project.c.project_name == f"{self._project_name}")
+
+        with self._engine.connect() as connection:
+            result = connection.execute(stmt).fetchone()
+            self._project_hash = result[0]
+
+    def get_ontology(self) -> Dict:
+        active_project = Table("active_project", self._metadata, autoload_with=self._engine)
+
+        stmt = select(active_project.c.project_ontology).where(active_project.c.project_hash == f"{self._project_hash}")
+
+        with self._engine.connect() as connection:
+            result = connection.execute(stmt).fetchone()
+            ontology = result[0]
+
+        return ontology
```

Comment:
Would be better to return an ontology structure?
```suggestion
        return OntologyStructure.from_dict(ontology)
```
Remember the import